client/src/components/restaurant/restaurant.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Tab, Tabs, Container } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Api from '../../services/api';
+import About from './about';
+import '../../style/restaurant.css';
+
+class Restaurant extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurant: {},
+      isFetching: false,
+    };
+  }
+
+  async componentDidMount() {
+    this.getOne();
+  }
+
+  getOne() {
+    const { match } = this.props;
+    Api.getOne('restaurants', match.params.id)
+      .then((response) => {
+        if (response.error) {
+          // eslint-disable-next-line no-console
+          console.error(response);
+          return;
+        }
+        this.setState({
+          restaurant: response.data,
+          isFetching: true,
+        });
+      });
+  }
+
+  render() {
+    const { isFetching, restaurant } = this.state;
+    return (
+      <Container>
+        <h2>{restaurant.name}</h2>
+        <Tabs defaultActiveKey="about">
+          <Tab eventKey="about" title="About">
+            <About restaurant={restaurant} isFetching={isFetching} />
+          </Tab>
+          <Tab eventKey="menu" title="Menu">
+            <h3>Menu</h3>
+          </Tab>
+          <Tab eventKey="events" title="Events">
+            <h3>Events</h3>
+          </Tab>
+          <Tab eventKey="feedback" title="Feedbac">
```

Comment:
"title" should be "Feedback", a little definition mistake

Reply:
I guess we need something like **Tab**-"wrapper" (or even **RestaurantTab**) component with props due to React.js. It will also help our code to be easier.

Reply:
Sorry, I'm not sure that understand you

Reply:
@narekacy Could you explain it better?

Reply:
Check this [video tutorial](https://www.youtube.com/watch?v=-mDs48HB3II&list=PLcvhF2Wqh7DNVy1OCUpG3i5lyxyBWhGZ8&index=16), please. You may set 1.5 or even 2.0 playback speed, this guy explains it in excellent way. 

When you finish watching it, remind yourself that you also can use an array to pass in it your component as props with `map()` magic.